charts/model-engine/values_sample.yaml

```diff
@@ -91,6 +91,14 @@ autoscaling:
   prewarming:
     enabled: false
 
+# for async endpoints, Celery autoscaler scales the number of pods based on number of requests
+# message_broker should be either "sqs" or "elasticache"
+# num_shards is number of instances of the autoscaler
+celery_autoscaler:
+  enabled: true
+  message_broker: sqs
```

Comment:
this is already defined as the root-level "messageBrokerType" (can see that at the bottom of this file)